generators/client/templates/angularjs/gulp/_copy.js

```diff
@@ -84,7 +84,13 @@ function fonts() {
 }
 
 function common() {
-    return gulp.src([config.app + 'robots.txt', config.app + 'favicon.ico', config.app + '.htaccess'], { dot: true })
+    return gulp.src([
+        config.app + 'robots.txt',
+        config.app + 'favicon.ico',
+        config.app + '.htaccess',
+        config.app + 'manifest.webapp',
+        config.app + 'sw.js'
```

Comment:
done. @sendilkumarn can you add some documentation for this. Maybe  new page on PWA with instructions on how to enable it